examples/go-target/main.go

```diff
@@ -0,0 +1,26 @@
+// This is the target program that we will attach the uprobe to. It is a simple
+// program that runs the `ls` command every 5 seconds and logs the output.
```

Comment:
This doesn't seem to be running ls anymore

Reply:
Good catch.  I started with ls, but changed it to call a local function.  The time also changed.  I'll correct the comment.